acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -206,20 +216,27 @@ export default async function init(element) {
   widgetContainer.className = `fsw wapper-${VERB}`;
   widget.appendChild(widgetContainer);
 
-  const isReturningUser = window.localStorage.getItem('pdfnow.auth');
   const isRedirection = /redirect_(?:conversion|files)=true/.test(window.location.search);
-  const preRenderDropZone = !isReturningUser && !isRedirection;
+  let isLimitExhausted = false;
+  const isExportVerb = VERB.includes('pdf-to');
+  const isCreateVerb = VERB.includes('to-pdf') || VERB === 'createpdf';
```

Comment:
don't need check for createpdf here and in L117. Either one will do

Reply:
we need this because 'createpdf' is the verb name (not group) for pdf converter and 'to-pdf' captures the remainder of create verb names

Reply:
then you don't need L117?

Reply:
if `VERB==="createpdf"` even if `isCreateVerb` is true you'll never make it to L222

Reply:
agreed, needed to remove L117. Updated